docs/source/data/resolvers.mdx

```diff
@@ -461,6 +461,16 @@ type Author {
 
 If the resolver for the `books` field returns an array of objects that each contain a `title` field, then you can use a default resolver for the `title` field. The default resolver will correctly return `parent.title`.
 
+## Resolving unions and interfaces
+
+There are GraphQL types that enable you to define a field that returns one of multiple possible object types (i.e. [unions and interfaces](../schema/unions-interfaces/#resolving-a-union)). To resolve a field that can return different object types, you must define a `__resolveType` function to inform Apollo Server _which_ object type to return.
```

Comment:
Ah two more things:

* comma after "i.e." (standard convention)
* I think "_which_ object type to return" isn't quite right. I think more accurate is "_which_ type of object is being returned".

The distinction is super subtle: the field resolver for a field that returns a union or interface can execute just like any other resolver, and get back an object via whatever logic is appropriate. But then Apollo Server needs to be told which type that object is, so that it knows which _sub_-resolvers to execute for that object's fields (and so it can populate `__typename` I reckon).

Reply:
That is a great point and distinction to make!